.deepsource.toml

```diff
@@ -0,0 +1,8 @@
+version = 1
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
I think it's not necessary since we use LGTM for continuous review. :)

Reply:
Hey @dwisiswant0 thanks for your response.😀 I'm removing `.deepsource.toml` file if you don't want to try out DeepSource. 

However, I want to highlight a few points which explain how DeepSource wins over LGTM:
- DeepSource [Python analyzer](https://deepsource.io/directory/analyzer/python/) detects ~558 issues [set of rules] while LGTM covers only [148 rules](https://lgtm.com/projects/g/dwisiswant0/apkleaks/queries/?pack=com.lgtm%2Fpython-queries). 
- DeepSource offers an [Autofix feature ](https://deepsource.io/blog/autofix-future-of-code-reviews/)with which you can fix issues automatically in a couple of clicks. There are 88 Autofix available in Python analyzer. LGTM does not offer any sort of this feature.
- DeepSource offers [auto code formatter feature](https://deepsource.io/docs/transformer/), which helps auto transform your codebase to maintain code styling. Available for Python too. 🐍
- DeepSource guarantees less than 5% false positives.
- Much better integration and developer experience.